src/main/resources/avro/readmethods.avdl

```diff
@@ -72,7 +72,20 @@ record SearchReadsResponse {
 }
 
 /**
-Gets a list of `ReadAlignment` matching the search criteria.
+Gets a list of `ReadAlignment`s for one or more `ReadGroup`s.
+
+`searchReads` operates over a genomic coordinate space of reference sequence
+and position defined by the `Reference`s to which the requested `ReadGroup`s are
+aligned.
+
+If a target positional range is specified, search returns all reads whose
+alignment to the reference genome *overlap* the range. A query which specifies
+only read group IDs yields all reads in those read groups, including unmapped
+reads.
+
+All reads returned (including reads on subsequent pages) are ordered by genomic
+coordinate (by reference sequence, then position). Reads with equivalent genomic
+coordinates are returned in an unspecified but deterministic order.
```

Comment:
I see, this is a good point. How about we split this into two sentences then? "..are returned in an unspecified order. This order is required to be consistent with a server implementation, however, such that the list of results obtained from the same query is always the same". Or something along those lines?